src/Microsoft.PowerShell.Commands.Utility/Microsoft.PowerShell.Commands.Utility.csproj

```diff
@@ -35,6 +35,7 @@
     <Compile Remove="commands\utility\ShowCommand\ShowCommandParameterType.cs" />
     <Compile Remove="commands\utility\ShowCommand\ShowCommandProxy.cs" />
     <Compile Remove="commands\utility\update-list.cs" />
+    <Compile Remove="commands\utility\Unblockfile.cs" Condition=" '$(Configuration)' == 'Linux' " />
```

Comment:
I don't really have a preference although `csproj` approach seems more discoverable. However in this case, I don't think `Unblockfile.cs` should be completely excluded from Unix build as it will increase the number of `#if unix/endif` blocks in places where the code from this file is used, which will impact readability.